sdk/nextjs/src/pages/appWithDevCycle.tsx

```diff
@@ -87,21 +39,25 @@ export const appWithDevCycle = <Props extends NextJsAppProps>(
             <DevCycleProvider
                 config={{
                     sdkKey: devcycleSSR.sdkKey,
+                    user: devcycleSSR.user,
                     options: {
                         ...additionalOptions,
+                        ...(onServerside
```

Comment:
can this be on top of additional options? if the user defines the `disableAutomaticEventLogging` option it'll get overwritten here

Reply:
it has to be overwritten on the serverside, because event logging there isn't supported

Reply:
ah okay I see it won't override it if the onServerSide is false on the client now

Reply:
yup